homeassistant/components/medcom_ble/sensor.py

```diff
@@ -0,0 +1,124 @@
+"""Support for Medcom BLE radiation monitor sensors."""
+from __future__ import annotations
+
+import logging
+
+from medcom_ble import MedcomBleDevice
+
+from homeassistant import config_entries
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.const import PERCENTAGE, EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import CONNECTION_BLUETOOTH, DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+)
+
+from .const import DOMAIN, UNIT_CPM
+
+_LOGGER = logging.getLogger(__name__)
+
+# Note: no medcom device supports battery readings at the moment, but this
+# could be the case in the future.
```

Comment:
Please remove items that aren't supported yet. They can be added in the future if/when they become supported